arm-compute/2017-03-30/swagger/runCommands.json

```diff
@@ -198,6 +198,13 @@
           "type": "string",
           "description": "The run command id."
         },
+        "script": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The script to be executed."
```

Comment:
well excellent, that's the kind of information that belongs into the *documentation*! Please add it, what you just told me is supposed to be clear to *customers* from looking at the Swagger.